src/components/shared/cellRenderers/cellRenderer.scss

```diff
@@ -56,27 +72,21 @@ $iconSize: 14px;
     .pcs-renderer-text { color: themed('colorCellRendererText'); }
 
     &.highlight {
-
       .pcs-renderer-icon svg {
         fill: themed('colorCellRendererSvgFillHighlight');
 
         .disabled-icon { stroke: themed('colorCellRendererSvgFillHighlight'); }
       }
 
       .pcs-renderer-text { color: themed('colorCellRendererTextHighlight'); }
+    }
 
+    .glimmer-icon {
```

Comment:
Since this only has one child you can inline this. `.glimmer-icon svg { fill: themed('colorGlimmerSvgFill'); }`